CMakeLists.txt

```diff
@@ -1,14 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(mara_moveit_tutorials)
 
-add_compile_options(-std=c++11)
-
-find_package(Eigen3 REQUIRED)
```

Comment:
Why is this removed?

Reply:
I followed this one here: https://github.com/ros-planning/moveit_tutorials/blob/master/CMakeLists.txt

I don't think it's an essential change. I can put it as it was

Reply:
See https://github.com/ros-planning/moveit_tutorials/commit/c375fe742dc64e1354875bc06edafe7c0eae5dbb#diff-af3b638bc2a3e6c650974192a53c7291

Reply:
the line 6 has just changed position to line 24

Reply:
True that :+1: